### PR TITLE
Update image instructions

### PR DESCRIPTION
**Pull request by steffilazerte**
I update the general image instructions, specifically:
- details on image alignment
- switching order so that alt text is discussed before examples
- simplifying example names 
- adding an example of html escaping

### CODE REVIEW

**Comment by steffilazerte** [comment]
@stefaniebutland Ready for review!

File: authortechnical.Rmd [hunk 1]
```diff
@@ -132,7 +132,7 @@ Refer [to this tutorial for details on what should go in alternative text](https
 - Control image **placement** with `class`  
   `{{< figure src = "image-name.png" alt = "informative description" class = "center" >}}`
     - `pull-left` - Left-align the picture and wrap text around it
-    - `center` - Centre the picture (no text wrapping)
+    - `center` - Center the picture (no text wrapping)
```
Comment (maelle) [inline authortechnical.Rmd:135]: lol

Comment (maelle) [reply authortechnical.Rmd:135]: (I'm sorry for the typo, of course!!)

**Review by stefaniebutland — APPROVED** [review]
LGTM. Thank you @steffilazerte & @maelle

File: authortechnical.Rmd (hunk 1 above)
Comment (steffilazerte) [reply authortechnical.Rmd:135]: Oh that's my typo :grin: I'm Canadian, so I have to force myself to use American spelling with rOpenSci (it hurts!)

**Pull request merged**